discord/guild.py

```diff
@@ -789,6 +790,10 @@ def created_at(self):
         """:class:`datetime.datetime`: Returns the guild's creation time in UTC."""
         return utils.snowflake_time(self.id)
 
+    def is_nsfw(self):
+        """:class:`bool`: Checks if the guild is NSFW."""
+        return self.nsfw
+
```

Comment:
I think instead of having a function for this the attribute should just be documented. The function historically was because channel nsfw checks were a bit more complicated before the invention of the attribute, so you can see it as legacy. 

If the attribute is added and documented it should have the versionadded attribute for 2.0.

Reply:
Thank you for the feedback. It is now only an attribute.